react/index.d.ts

```diff
@@ -606,165 +606,174 @@ declare namespace React {
         onTransitionEndCapture?: TransitionEventHandler;
     }
 
+    // See CSS 3 CSS-wide keywords https://www.w3.org/TR/css3-values/#common-keywords
+    // See CSS 3 Explicit Defaulting https://www.w3.org/TR/css-cascade-3/#defaulting-keywords
+    // "all CSS properties can accept these values"
+    type CSSWideKeywords = "initial" | "inherit" | "unset";
```

Comment:
An instance of `CSSWideKeywords` is a *single* keyword. The type should be singular to avoid confusion, as a user might interpret the pluralization to indicate the instance is an array.